src/seedu/addressbook/parser/Parser.java

```diff
@@ -53,6 +54,8 @@
  */
 public class Parser {
 
+    public static final int PRICE_STRING_MAX_LENGTH = 5;
```

Comment:
Too short
The price String is in the format $A.BC
For example: "$5.00" has the length of 5

Reply:
Recommendation: if the max price cannot exceed $10000.00, then max length should be 8 (length of $9999.99)